packager/packager.cc

```diff
@@ -1838,6 +1840,14 @@ vector<ast::ParsedFile> Packager::findPackages(core::GlobalState &gs, WorkerPool
                 } else {
                     packages.db.enterPackage(move(pkg));
                 }
+
+                if (gs.runningUnderAutogen) {
+                    it = files.erase(it);
```

Comment:
Drop all package files in autogen (after packages are entered), to prevent namer errors.

Reply:
FYI erase like this is O(total len * deletes).  It's why `remove_if` exists.

Reply:
Ah ok, wanted to do it in-place in the same loop, but that makes sense. Will fix.